### DISCUSSION TITLE
notification dont show when use swoole package

### DISCUSSION THREAD

**Discussion by farhad-rohani** [filamentphp/filament#2343] [Help]
### Package

filament/filament

### Package Version

v2.10.32

### Laravel Version

v9.5.1

### Livewire Version

v2.10.4

### PHP Version

PHP 8.1.5

### Bug description

I use swoole package for increase performance php.
when create model in filament notification show.
but when edit the model notification don't show.

### Relevant log output

_No response
![Screenshot from 2022-05-08 16-12-36](https://user-images.githubusercontent.com/9474884/167314230-c00a6bea-b529-4771-96ce-a59f8cf89e4b.png)
_

**Comment by farhad-rohani** [comment]
notification show when create model 
and redirect from create page to edit page

**Comment by danharrin** [comment]
Duplicate of #2306
